config.go

```diff
@@ -45,30 +40,8 @@ func (c Config) getGraylogAppName() string {
 	return appName
 }
 
-func (c Config) getGraylogEnv() (int, error) {
-	if c._isMock {
-		return c._mockEnv, c._mockEnvError
-	}
-
-	// If we're running test return test logger env.
-	if flag.Lookup("test.v") != nil {
-		return testEnv, nil
-	}
-
-	envLevelString := os.Getenv("GRAYLOG_ENV")
-	if envLevelString == "" {
-		return 0, errors.New(envNotSetErrorString)
-	}
-
-	envLevel, err := strconv.Atoi(envLevelString)
-	if err != nil {
-		return 0, errors.New("could not properly parse GRAYLOG_ENV")
-	}
-
-	return envLevel, nil
-}
-
-func (c Config) getGraylogHandlerType() graylog.Transport {
+func (e *EnvConfig) getGraylogHandlerType() graylog.Transport {
+	defaultHandlerType := tlsTransport
 	handlerType := os.Getenv("GRAYLOG_HANDLER_TYPE")
 	if handlerType == "" {
```

Comment:
Wouldn't this still panic regardless of the default, if GRAYLOG_HANDLER_TYPE is not set?

Reply:
Oh my, that was a dumb mistake, good catch.

Reply:
@ian-axelrod fixed